src/Form/input-classifiers.test.js

```diff
@@ -237,28 +252,32 @@ afterAll(() => {
 
     testResults.forEach((result) => {
         result.scores.forEach((field) => {
-            if (!totalFieldsByType[field.manualScore]) {
-                totalFieldsByType[field.manualScore] = 0
-                totalFailuresByFieldType[field.manualScore] = 0
+            const {manualScore, inferredType} = field
+            if (!totalFieldsByType[manualScore]) {
+                totalFieldsByType[manualScore] = 0
+                totalFailuresByFieldType[manualScore] = 0
             }
 
-            if (field.manualScore !== field.inferredType) {
+            if (manualScore !== inferredType) {
                 totalFailedFields++
-                totalFailuresByFieldType[field.manualScore]++
+                totalFailuresByFieldType[manualScore]++
             }
-            if (field.manualScore === 'unknown' && field.inferredType !== field.manualScore) {
+            if ((!manualScore || manualScore === 'unknown') && inferredType !== manualScore) {
                 totalFalsePositives++
             }
             totalFields++
-            totalFieldsByType[field.manualScore]++
+            totalFieldsByType[manualScore]++
         })
     })
 
     console.log(
         'Input classification statistics:',
-        '\n% of failing sites:\t\t' + Math.round(proportionFailingSites * 100) + '%',
-        '\n% of failing fields:\t' + Math.round((totalFailedFields / totalFields) * 100) + '%',
-        '\n% of false positive fields:\t' + Math.round((totalFalsePositives / totalFields) * 100) + '%',
+        '\n% of failing sites:\t\t' + (proportionFailingSites * 100).toFixed(1) + '%',
+        '\n\t\t (' + failingSites + ' of ' + allSites + ', for a total of ' + testCases.length + ' forms)',
+        '\n% of failing fields:\t' + ((totalFailedFields / totalFields) * 100).toFixed(1) + '%',
+        '\n\t\t (' + totalFailedFields + ' of ' + totalFields + ')',
+        '\n% of false positive fields:\t' + ((totalFalsePositives / totalFields) * 100).toFixed(1) + '%',
+        '\n\t\t (' + totalFalsePositives + ' of ' + totalFields + ')',
```

Comment:
Just a bunch of improvements in how we display the data, which now looks like this:

```
Input classification statistics:
% of failing sites:         17.4%
     (20 of 115 for a total of 239 forms)
% of failing fields:        3.1%
     (38 of 1209)
% of false positive fields: 0.5%
     (6 of 1209)
```